XLS-0085d-token-escrow/README.md

```diff
@@ -47,7 +47,6 @@ The `EscrowCreate` transaction is modified as follows:
 | Field     | Required? | JSON Type        | Internal Type | Description                                                                                                                                        |
 |-----------|-----------|------------------|---------------|----------------------------------------------------------------------------------------------------------------------------------------------------|
 | `Amount`  | Yes       | Object or String | Amount  | The amount to deduct from the sender's balance and and set aside in escrow. Once escrowed, this amount can either go to the Destination address (after any `Finish` times/conditions) or returned to the sender (after any cancellation times/conditions). Can represent [XRP, in drops](https://xrpl.org/docs/references/protocol/data-types/basic-data-types#specifying-currency-amounts), an [IOU](https://xrpl.org/docs/concepts/tokens/fungible-tokens#fungible-tokens) token, or an [MPT](https://xrpl.org/docs/concepts/tokens/fungible-tokens/multi-purpose-tokens). Must always be a positive value.|
-| `CancelAfter`  | False       | Number | UInt32        | (Optional) The time, in seconds since the Ripple Epoch, when this escrow expires. This value is immutable; the funds can only be returned to the sender after this time. Required when creating an Escrow with IOU or MPT |
```

Comment:
Clarification: Does this mean that `CancelAfter` field is not mandatory for Escrow's with IOU (or) MPT Amounts?

Reply:
Currently yes

Reply:
@dangell7 Can you clarify the removal of this field in this PR? 

From my perspective, it looks like you're simply signfying that there's no change in the `CancelAfter`, but clarify that if I'm interpreting this change incorrectly.